Sources/FluentKit/Migration/Migration.swift

```diff
@@ -1,6 +1,19 @@
+/// `Migration` can assist with database migration. It can choose
+/// to create a new database schema for migration, or drop an
+/// existing schema. These actions are done in-place, returning no
+/// values.
 public protocol Migration {
     var name: String { get }
+    /// Prepares new database schema for the migration.
+    /// - Parameters:
+    ///     - database: `Database` to be created.
```

Comment:
Tech nit - this is the database to run the migration on not the database to be created